storage/innobase/xtrabackup/src/changed_page_tracking.cc

```diff
@@ -145,6 +145,24 @@ void deinit(xb_space_map *space_map) {
   }
 }
 
+/* Get the start page-tracking-lsn
+@param[in] connection               MySQL connectionn
+@return the page tracking start lsn */
+lsn_t get_pagetracking_start_lsn(MYSQL *connection) {
+  std::string udf_get_start_lsn = Backup_comp_constants::udf_get_start_lsn;
+  char *start_lsn_str = read_mysql_one_value(
+      connection, ("SELECT " + udf_get_start_lsn + "()").c_str());
+  lsn_t backup_start_lsn = strtoll(start_lsn_str, nullptr, 10);
```

Comment:
can you please use different name other than `backup_start_lsn`? may be page_track_start_lsn?

Reply:
fixed